core-bundle/tests/Functional/RoutingTest.php

```diff
@@ -776,14 +776,14 @@ public function getRootAliasesWithLocale(): array
             'Redirects to the language root if one of the accept languages matches' => [
                 '/',
                 301,
-                'Redirecting to en/',
+                'Redirecting to http://root-with-index.local/en/',
                 'en,de',
                 'root-with-index.local',
             ],
             'Redirects to the language fallback if none of the accept languages matches' => [
                 '/',
                 301,
-                'Redirecting to en/',
+                'Redirecting to http://root-with-index.local/en/',
```

Comment:
This change is wrong and makes the tests fail. 

```suggestion
                'Redirecting to en/',
```